### DISCUSSION TITLE
Manual Mesh Levelling

### DISCUSSION THREAD

**Discussion by wbvczar** [Jyers/Marlin#69] [Q&A] [answered]
I love this new firmware and it is working great for me.  I was wondering, if I have performed the manual mesh bed levelling and stored it to eeprom, will I have to re-run the levelling when I upgrade to the 1.1 firmware?  Or will it pull it from the eeprom?

**Comment by Schnabulation** [comment, marked as answer]
You should always clear EEPROM after flashing a new firmware so it does not retain false value.

**Comment by Jyers** [comment]
As Schnabulation said, while it may retain the levelling data, it is best to reset EEPROM after a flash as leaving it with old data can cause unexpected behavior.
